azurerm/internal/services/authorization/role_assignment_resource.go

```diff
@@ -197,19 +222,19 @@ func resourceArmRoleAssignmentCreate(d *pluginsdk.ResourceData, meta interface{}
 		properties.RoleAssignmentProperties.PrincipalType = authorization.ServicePrincipal
 	}
 
-	if err := pluginsdk.Retry(d.Timeout(pluginsdk.TimeoutCreate), retryRoleAssignmentsClient(d, scope, name, properties, meta)); err != nil {
+	if err := pluginsdk.Retry(d.Timeout(schema.TimeoutCreate), retryRoleAssignmentsClient(d, scope, name, properties, meta, tenantId)); err != nil {
```

Comment:
this should be `pluginsdk.TimeoutCreate`, I will submit a PR to fix it soon

Reply:
ty i was trying to resolve it via github - didn't go as well as i'd hoped 😓